.github/workflows/api.yaml

```diff
@@ -23,6 +23,19 @@ jobs:
         with:
           version: 8
           run_install: false
+          
+      - name: Get pnpm store directory
```

Comment:
I think this one in here is out of alignment

Reply:
got it!